src/iocore/net/QUICNetVConnection.cc

```diff
@@ -645,8 +645,8 @@ QUICNetVConnection::_handle_read_ready()
     QUICConVDebug("stream %" PRIu64 " is readable\n", s);
     QUICStream *stream = static_cast<QUICStream *>(this->_stream_manager->find_stream(s));
     if (stream == nullptr) {
-      this->_stream_manager->create_stream(s);
-      stream = static_cast<QUICStream *>(this->_stream_manager->find_stream(s));
+      [[maybe_unused]] QUICConnectionError err;
```

Comment:
Eventually we will check for `err`.

Reply:
Maybe you should check it with ink_assert for now.

Reply:
We'd never discard a value that we use on the following normal flow. Returning an error with `[[nodiscard]]` attribute would be nicer in general, although I'm not sure what error can happen in `create_stream`.

Reply:
The only error could be that there is no app for that particular stream id, but seems unlikely. The only reason I kept the error was because it was in the original API.

We can just rely on the returned vale, either a new stream or  `nullptr` and remove the error from this API.

Reply:
The API has that signature because it was designed before we start using Quiche. The old implementation had to check the number of streams open and return an error if the number exceeds the limit. And I thought returning a QUICConnectionError would be handy because the error code should be always the same if stream creation fails.

Reply:
How strong you feel about the old API? If I put the old API back, then we won't be able to get the stream when creating a new one which will involve `find_stream` after creating it(as before).

Reply:
Oh, I didn't mean we should keep the original API. I was just telling the background. The only debatable point would be which should be the return type of create_stream, the error or the stream. But it's not a huge deal. I'm fine with the current API.